docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -241,6 +287,27 @@ function uiowa_core_entity_view(array &$build, EntityInterface $entity, EntityVi
   }
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function uiowa_core_preprocess_node(&$variables) {
+  $admin_context = \Drupal::service('router.admin_context');
+  if (!$admin_context->isAdminRoute()) {
+    // Get the current user.
+    $user = \Drupal::currentUser();
+    // Showing a message to viewers and up about how scheduling locks
+    // editors from editing the content for content with moderation.
+    if ($user->hasPermission('view any unpublished content') === TRUE) {
+      if (isset($variables['publish_on']) && $variables['node']->moderation_state?->value) {
+        if ($variables['view_mode'] === 'full') {
+          \Drupal::messenger()
+            ->addStatus(t('Due to scheduling, editors are restricted from editing until published.'));
```

Comment:
```suggestion
            ->addStatus(t('This content has been scheduled for publishing and editors are restricted from editing until published.'));
```

Reply:
Some redundancy with the "`node:title` is scheduled to be published `node:publish_on`" message that may also appear at the same time but I am fine with this.